io.c

```diff
@@ -7631,26 +7631,20 @@ rb_open_file(int argc, const VALUE *argv, VALUE io)
     return io;
 }
 
-
 /*
  *  Document-method: File::open
  *
  *  call-seq:
- *     File.open(filename, mode="r" [, opt])                 -> file
- *     File.open(filename [, mode [, perm]] [, opt])         -> file
- *     File.open(filename, mode="r" [, opt]) {|file| block } -> obj
- *     File.open(filename [, mode [, perm]] [, opt]) {|file| block } -> obj
+ *    File.open(path, mode = 'r', perm, **opts]) -> file
+ *    File.open(path, mode = 'r', perm, **opts]) {|f| ... } -> object
```

Comment:
This call-seq appears incorrect, because `perm` is not a required argument: `File.open('ruby.c')`. `perm` should default to `0666`, as that is what Ruby uses internally.